core/src/main/java/org/stellar/anchor/sep31/Sep31TransactionBuilder.java

```diff
@@ -54,16 +55,23 @@ public Sep31TransactionBuilder amountOutAsset(String amountOutAsset) {
     return this;
   }
 
+  @Deprecated
   public Sep31TransactionBuilder amountFee(String amountFee) {
     txn.setAmountFee(amountFee);
```

Comment:
I think this can be a bug. While `setAmountFee` is still supported (though deprecated), the it will be inconsistent with `feeDetails` field. 

Same inconsistency may exist in Sep6 and Sep24 builders.

Reply:
Good catch. In all places `fee` and `feeAsset` were derived from `feeDetails`, but it could've created a bug in the future.
I updated the builder.
For transaction updates it's still possible to update fee and `feeAsset` independently, but RPC handlers validate that either `fee` + `feeAsset` or `feeDetails` are updated